### PR TITLE
Alert branch 1

### PR DESCRIPTION
**Pull request by ruchiragoswami**

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    margin-left: 1rem;
+}
+
+.warning{
+    background-color: var(--warning-color);
+}
+
+
+.success {
+    background-color: #D1FAE5;
```
Comment (lalith96) [inline alert/alert.css]: Here color is hardcoded. You can use variable.

**Review by lalith96 — COMMENTED** [review]
Everything is fine except color is hardcoded in alert.css.

File: alert/alert.css (hunk 1 above)
Comment (ruchiragoswami) [reply alert/alert.css]: Thanks. Fixed now :)

**Review by lalith96 — APPROVED** [review]

**Review by ScoobyDubeyDo — APPROVED** [review]

**Review by Tarandeep-s1ngh — APPROVED** [review]
Looks good to me! Approved.

**Pull request merged**